phpunit.xml.dist

```diff
@@ -1,12 +1,13 @@
-<phpunit bootstrap="phpunit.php">
-    <testsuites>
-        <testsuite name="atlas/table">
-            <directory>tests/</directory>
-        </testsuite>
-    </testsuites>
-    <filter>
-        <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">./src</directory>
-        </whitelist>
-    </filter>
+<?xml version="1.0"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="phpunit.php" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```

Comment:
This conflicts with the allowed versions in the Composer configuration, 7 and 9.

The local schema file can be used instead:
```xml
xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
```